kube/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM apache/spark:3.4.2
+
+USER root
+
+# Installing JDK11 as the image comes with JRE
+RUN apt update \
+    && apt install -y git \
+    && apt install -y curl \
+    && apt install -y openjdk-11-jdk \
+    && apt clean
+
```

Comment:
Doesn't `apache/spark:3.4.2` have Java installed?

Reply:
I think it has JRE but not JDK based on the comment

Reply:
Yeah, its only JRE but the folder name is confusing and is said it is openjdk, but in fact no javac in the bin folder